gramps/gui/editors/editdate.py

```diff
@@ -74,13 +74,15 @@
 #
 #-------------------------------------------------------------------------
 MOD_TEXT = (
-    (Date.MOD_NONE, _('Regular')),
-    (Date.MOD_BEFORE, _('Before')),
-    (Date.MOD_AFTER, _('After')),
-    (Date.MOD_ABOUT, _('About')),
-    (Date.MOD_RANGE, _('Range')),
-    (Date.MOD_SPAN, _('Span')),
-    (Date.MOD_TEXTONLY, _('Text only')))
+    (Date.MOD_NONE       , _('Regular')),
+    (Date.MOD_BEFORE     , _('Before')),
+    (Date.MOD_AFTER      , _('After')),
+    (Date.MOD_ABOUT      , _('About')),
+    (Date.MOD_RANGE      , _('Range')),
+    (Date.MOD_FROM       , _('From')),
```

Comment:
Why did you add spaces ? It's nicer but pylint says it's not good:

$  pylint3 gramps/gui/editors/editdate.py
Using config file /home/gramps/pull-gramps-1/.pylintrc
************* Module gramps.gui.editors.editdate
C: 77, 0: Exactly one space required after comma
    (Date.MOD_NONE,    _('Regular')),
                  ^ (bad-whitespace)

Reply:
Fixed.  Thank you.